### PR TITLE
Assignment_2.6

### PR DESCRIPTION
**Pull request by saachi-tiger**
Fixes issue #21 , which includes:
- Create ML workflow scripts
- Use the argparse module to accept user arguments when running the scripts.
- Setup three jobs in a single workflow which does Unit test job, Build job and Deploy job

### CODE REVIEW

File: .github/workflows/build_script_2.6.yml [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+          source $HOME/miniforge3/etc/profile.d/conda.sh
+          conda activate mle-dev
+          pip install dist/*.whl  # Reference the correct path for the .whl files
+
+      - name: Run the scripts in help mode
+        run: |
+          source $HOME/miniforge3/etc/profile.d/conda.sh
+          conda activate mle-dev
+          python scripts/train.py -h
+          python scripts/score.py -h
```
Comment (abilashravi-ta) [inline .github/workflows/build_script_2.6.yml]: All the 3 scripts must be run in both the modes (_help mode_ & _normal mode_).

File: scripts/ingest_data.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import argparse
+from house_package_saachi.data_ingestion import fetch_housing_data, load_housing_data
```
Comment (abilashravi-ta) [inline scripts/ingest_data.py:2]: You can delete the `main.py` file because these 3 scripts are there to replace it.

**Review by abilashravi-ta — CHANGES_REQUESTED** [review]
@saachi-tiger just make the following changes.. Rest of them looks fine.

File: .github/workflows/build_script_2.6.yml (hunk 1 above)
Comment (saachi-tiger) [reply .github/workflows/build_script_2.6.yml]: Added script to run in normal mode

File: scripts/ingest_data.py (hunk 2 above)
Comment (saachi-tiger) [reply scripts/ingest_data.py:2]: Removed main.py file

**Review by abilashravi-ta — APPROVED** [review]

**Pull request merged**